NBitcoin/PubKey.cs

```diff
@@ -29,7 +29,7 @@ public enum ScriptPubKeyType
 		/// </summary>
 		SegwitP2SH
 	}
-	public class PubKey : IBitcoinSerializable, IDestination, IComparable<PubKey>, IEquatable<PubKey>
+	public class PubKey : IBitcoinSerializable, IDestination, IComparable<PubKey>, IEquatable<PubKey>, IComparable

```

Comment:
`IComparable` makes no sense for `PubKey`.

Reply:
I think it makes sense to follow [bip67](https://github.com/bitcoin/bips/blob/master/bip-0067.mediawiki)

Reply:
ACK.